scripts/zones/Southern_San_dOria/npcs/Varchet.lua

```diff
@@ -28,15 +28,13 @@ function onTrade(player,npc,trade)
         local pdie2 = math.random(1,6);
         local ptotal = pdie1 + pdie2;
 
-           if (ptotal > vtotal) then
-              player:startEvent(0x0207,vdie1,vdie2,vtotal,pdie1,pdie2,ptotal,GAME_WON);
-           elseif (vtotal > ptotal) then
-              player:startEvent(0x0207,vdie1,vdie2,vtotal,pdie1,pdie2,ptotal,GAME_LOST);
-           elseif (ptotal == vtotal) then
-              player:startEvent(0x0207,vdie1,vdie2,vtotal,pdie1,pdie2,ptotal,GAME_TIE);
-           else
-              printf("Results Varchet's Roll : %u, Player's Roll : %u", vtotal, ptotal);
```

Comment:
I went ahead and removed the extra else since it's not possible for an "else" to occur here. You can only win, lose, or tie. There is no fourth possibility.